tests-functional/clients/status_backend.py

```diff
@@ -63,9 +63,9 @@ def __init__(self, await_signals=[], privileged=False):
     def _start_container(self, host_port, privileged):
         docker_project_name = option.docker_project_name
 
-        timestamp = int(time.time() * 1000)  # Keep in sync with run_functional_tests.sh
+        git_sha = os.popen("git rev-parse --short HEAD").read().strip()
         image_name = f"{docker_project_name}-status-backend:latest"
-        container_name = f"{docker_project_name}-status-backend-{timestamp}"
+        container_name = f"{docker_project_name}-{git_sha}-status-backend-{host_port}"
```

Comment:
Can we rely on Jenkins build id? This is for sure unique.
Or even job name + build id.
https://ci.status.im/env-vars.html/

This will not work on dev machines, where git sha should be enough.

Reply:
I modified approach to use BUILD_ID when CI env var is set, else git sha.
This should work in CI and locally.